doc/tutorials/mount.md

```diff
@@ -1,56 +1,146 @@
-Elektra provides a global key database where configuration of all
-applications can be integrated.
+# Mounting #
 
-There are different forms of integration.  In this tutorial we will
-discuss a light form of integration that can be achieved without
-elektrifying applications.
+Elektra provides a global key database where configuration of all applications can be integrated.
 
-The idea is to manipulate the configuration values directly in the
-configuration files via the key database.  Thus the applications will
-read the configuration file, changes in the key database will be picked
-up by applications.
+There are different forms of integration. In this tutorial we will discuss a light form of integration that can be achieved without [elektrifying](/doc/help/elektra-glossary.md) applications.
```

Comment:
Good idea to link to glossary!

Do you think it is clear what "integration" means?